cmd/apprepository-controller/server/controller.go

```diff
@@ -136,7 +139,7 @@ func NewController(
 		UpdateFunc: func(oldObj, newObj interface{}) {
 			oldApp := oldObj.(*apprepov1alpha1.AppRepository)
 			newApp := newObj.(*apprepov1alpha1.AppRepository)
-			if oldApp.Spec.URL != newApp.Spec.URL || oldApp.Spec.ResyncRequests != newApp.Spec.ResyncRequests {
+			if !reflect.DeepEqual(oldApp.Spec, newApp.Spec) {
```

Comment:
@absoludity, do you know if there's any reason for checking the equality manually instead of just using `reflect.DeepEqual` as I'm proposing herein?  I guess that a change in the AppRepo spec should trigger an update in the controller, doesn't it?

Reply:
Nope. I can't see anything on that [AppRepo Spec struct ](https://pkg.go.dev/github.com/vmware-tanzu/kubeapps/cmd/apprepository-controller/pkg/apis/apprepository/v1alpha1#AppRepositorySpec) that, if changed, should not trigger an update. Agree 100%.